### ISSUE TITLE
Error upgrade from 9.1.1 to 9.8.1

### ISSUE THREAD

**Issue by markmcavoy** [dnnwerk/dnnScript#15]
We are trying to upgrade from 9.1.1 to 9.8.1 on portal we have inherited, but we keep getting an error about a foreign key constraint `FK_Portals_PortalGroups`

We have attempted the upgrade multiple times trying
 - TurboUnSchema911.sql
 - TurboUnSchema932.sql
 
However we get the same error either both attempts.

`Error: The UPDATE statement conflicted with the FOREIGN KEY constraint "FK_Portals_PortalGroups". The conflict occurred in database "val_dnn_update", table "dbo.PortalGroups", column 'PortalGroupID'. The statement has been terminated. at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)`

I can see the same error in this forum post, https://dnncommunity.org/forums/aft/1705 but it doesn't explain or shed any light on what the fix is.

**Comment by roman-yagodin** [comment]
Having similar problem during upgrade from DNN 8.0.4 to 9.8.1, it reveals itself during upgrade wizard run:

![Screenshot_2021-02-12 Upgrade](https://user-images.githubusercontent.com/5691178/108160545-54192100-70fa-11eb-9e9d-f4f52f29d30b.png)

Removing the `FK_Portals_PortalGroups` key fixed the issue for me.

I think removing it is perfectly safe, as there is no such key in the clean DNN 9.8.1 install.

**Comment by markmcavoy** [comment]
Yes that works perfectly, it was just that foreign key that wasn't removed the with the TurboUnSchema 

Thank you for sharing the tip!
